docs/user-guide/cli-scsplugin.md

```diff
@@ -24,7 +24,14 @@ For detailed command, actions, and option documentation for this plug-in, see ou
 
 Before you install the plug-in, meet the software requirements in [Software requirements for Zowe CLI plug-ins](cli-swreqplugins.md).
 
-**Note:** If you're using npm@7 or npm@8, an Internet connection is required during installation of the Secure Credential Store plug-in. Additionally, there is a prerequisite on Windows for npm@7 and npm@8 users: `npm install -g prebuild-install`
+**npm@7 and npm@8 Requirements:**
+
+ * Ensure that your computer is connected to the Internet when you install the Secure Credential Store plug-in.
+ * On Windows operating systems, issue the following command before you issue the command to install the Secure Credential Store plug-in:
```

Comment:
maybe just 

"...issue the following command before you install the Secure Credential Store plug-in:"
?

It sounds a bit repetitive with the word "command" mentioned twice :)